scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -646,10 +708,10 @@
       (is (= {"no-pool" (list (util/job-ent->map job-entity))}
              (sched/rank-jobs test-db offensive-job-filter))))))
 
-(deftest test-virtual-machine-lease-adapter
+(deftest test-virtual-machine-lease-adapter-mesos
```

Comment:
test-mesos-virtual-machine-lease-adapter

Reply:
done. and changed k8s version to `test-k8s-virtual-machine-lease-adapter`